pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -77,6 +77,13 @@ type Manager interface {
 	// and is consulted to achieve NUMA aware resource alignment among this
 	// and other resource controllers.
 	GetTopologyHints(*v1.Pod, *v1.Container) map[string][]topologymanager.TopologyHint
+
+	// GetCPUs implements the podresources.GetCPUs interface to provide allocated
```

Comment:
random nit comment:

```suggestion
	// GetCPUs implements the podresources.CPUsProvider interface to provide allocated
```

Reply:
it's a welcome nit! fixing

Reply:
this actually is in the parent PR whose commits I copied in this one to make it compile, so I'll fix on a separate commit.

Reply:
yes, this commit from https://github.com/kubernetes/kubernetes/pull/93243, that PR already got lgtm from @RenaudWasTaken. I updated comment there.